rpc/client.go

```diff
@@ -276,6 +276,9 @@ func (c *Client) Call(result interface{}, method string, args ...interface{}) er
 // The result must be a pointer so that package json can unmarshal into it. You
 // can also pass nil, in which case the result is ignored.
 func (c *Client) CallContext(ctx context.Context, result interface{}, method string, args ...interface{}) error {
+	if result != nil && reflect.TypeOf(result).Kind() != reflect.Ptr {
+		return fmt.Errorf("Expected nil or pointer for result, got %v", result)
```

Comment:
Please change this error message to lowercase. I'd also phrase it slightly differently, maybe "call result parameter must be pointer or nil interface".

Reply:
Done